build.gradle

```diff
@@ -218,6 +218,7 @@ project(':ambry-server') {
                 project(':ambry-replication')
         compile "io.dropwizard.metrics:metrics-core:$metricsVersion"
         compile "io.dropwizard.metrics:metrics-jmx:$metricsVersion"
+        compile "io.netty:netty-buffer:$nettyVersion"
```

Comment:
why was this dep added?

Reply:
It's for next PR, but let me remove it this PR.